backend/api/v1/endpoints/research.py

```
@@ -0,0 +1,373 @@
+                "details": paper.details,
+                "creator_id": paper.creator_id,
+            }
+            for paper in papers
+        ]
+
+    except Exception as e:
+        logging.error(f"Error fetching user papers: {str(e)}")
+        raise HTTPException(status_code=500, detail= give_error)
+    
+@router.get("/post_research_papers_others/")
+def get_other_research_papers(
```

Ruff (0.8.2)

Missing return type annotation for public function `get_other_research_papers`

(ANN201)